.travis.yml

```diff
@@ -28,10 +28,20 @@ matrix:
   exclude:
     - os: osx
       compiler: gcc
+
 before_install:
   - if [ `uname` = "Darwin" ]; then brew update; fi
+  - if [ `uname` = "Darwin" ]; then wget https://dl.bintray.com/jfrog/jfrog-cli-go/1.14.0/jfrog-cli-mac-386/jfrog; fi
```

Comment:
Might not be critical if we aren't using Darwin hosts as Travis controllers, but doesn't hurt.